### ISSUE TITLE
dependency installation doesn't work

### ISSUE THREAD

**Issue by jhumann** [AgBioData/Genome-Assembly-and-Annotation-Nomenclature_WG#2]
We have installed poetry (we don't have docker) on the server and I am trying to run the 'poetry install' command from the directory with the files from github and I get the following error:
[tool.poetry] section not found in /u/jhumann/Genome-Assembly-and-Annotation-Nomenclature_WG/pyproject.toml

**Comment by mpoelchau** [comment]
I get the same error as @jhumann (having installed pipx and poetry in a python environment). 

I also tried the docker route and get the following error:

```
(gaan-env) ARS5APVN2WD42YY:Genome-Assembly-and-Annotation-Nomenclature_WG mpoelchau$ docker build -t gaan-tool .
[+] Building 0.5s (7/9)                                                                                                    docker:desktop-linux
 => [internal] load build definition from Dockerfile                                                                                       0.0s
 => => transferring dockerfile: 520B                                                                                                       0.0s
 => [internal] load metadata for docker.io/library/python:3.8-slim                                                                         0.4s
 => [internal] load .dockerignore                                                                                                          0.0s
 => => transferring context: 106B                                                                                                          0.0s
 => CANCELED [1/5] FROM docker.io/library/python:3.8-slim@sha256:1d52838af602b4b5a831beb13a0e4d073280665ea7be7f69ce2382f29c5a613f          0.0s
 => => resolve docker.io/library/python:3.8-slim@sha256:1d52838af602b4b5a831beb13a0e4d073280665ea7be7f69ce2382f29c5a613f                   0.0s
 => => sha256:1d52838af602b4b5a831beb13a0e4d073280665ea7be7f69ce2382f29c5a613f 10.41kB / 10.41kB                                           0.0s
 => [internal] load build context                                                                                                          0.0s
 => => transferring context: 247B                                                                                                          0.0s
 => CACHED [2/5] WORKDIR /app                                                                                                              0.0s
 => ERROR [3/5] COPY poetry.lock pyproject.toml /app/                                                                                      0.0s
------
 > [3/5] COPY poetry.lock pyproject.toml /app/:
------
Dockerfile:8
--------------------
   6 |     
   7 |     # Copy just the poetry files (to leverage caching)
   8 | >>> COPY poetry.lock pyproject.toml /app/
   9 |     
  10 |     # Install dependencies using Poetry
--------------------
ERROR: failed to solve: failed to compute cache key: failed to calculate checksum of ref dfc062d9-67d8-4b3d-9fcc-9b047e4367a6::uz9u6tcwz233keunhylg8afe2: "/poetry.lock": not found
```

**Comment by adamjohnwright** [comment]
@jhumann I'm Sorry I didn't see this issue. It probably related to my second son's birth on April 2nd. I will be looking at this today and will let you know when I figure out what needs to change. @mpoelchau Thanks for pointing out this issue involving Docker. I will also let you know when your issue is solved. Hopefully later today.

**Comment by adamjohnwright** [comment]
@jhumand and @mpoelchau  Thanks again for pointing out these issues. Sorry about not getting to address them earlier. I have updated it so that it runs with just Poetry and inside of Docker. If you run into any more issues don't hesitate to make another ticket here. @mpoelchau you mentioned in an email to Ethy that the gene model name wasn't getting "." in between the sections of the name. This has now been fixed as well. Happy holidays.
